js/views/modals/moderatorDetails.js

```diff
@@ -0,0 +1,83 @@
+import loadTemplate from '../../utils/loadTemplate';
+import app from '../../app';
+import { followedByYou, followUnfollow } from '../../utils/follow';
+import BaseModal from './BaseModal';
+
+export default class extends BaseModal {
+  constructor(options = {}) {
+    const opts = {
+      removeOnClose: true,
+      ...options,
+    };
+
+    super(opts);
+    this.options = opts;
+    this.model = this.options.model;
```

Comment:
Can we ensure that the model is provided in the constructor? The view will bomb in places if it's not provided.